src/halodrops/processor.py

```diff
@@ -967,9 +986,32 @@ def add_q_and_theta_to_l2_ds(self):
         else:
             ds = self.l2_ds
 
-        ds = calc_q_from_rh(ds)
-        ds = calc_theta_from_T(ds)
+        ds = hh.calc_q_from_rh(ds)
```

Comment:
This should be a different commit. (that's my bad to not include `hh`, sorry :P)

Reply:
yes true. I subcontiously changed it at some point ^^'